pkg/archive/archive.go

```diff
@@ -395,6 +395,23 @@ func ReadSecurityXattrToTarHeader(path string, hdr *tar.Header) error {
 	return nil
 }
 
+// ReadUserXattrToTarHeader reads user.* xattr from filesystem to a tar header
+func ReadUserXattrToTarHeader(path string, hdr *tar.Header) error {
+	if xattrs, _ := system.Llistxattr(path); xattrs != nil {
```

Comment:
do we need to report errors != EOPNOTSUPP?

Reply:
Fixed in 0da4bc60b3babedb098dd7a75ae4ec0b75031eab.